docs/UserGuide.adoc

```diff
@@ -91,6 +91,50 @@ Expected output:
 
 *   `Entry deleted: 1. Laksa ($10.50) [Lunch][Food]`
 
+// tag::update[]
+=== Updating a Transaction’s details: `update`
+
+Format: `update i/INDEX | PREFIX_LETTER/UPDATED_DETAILS`
+
+****
+* Updates the Transaction's details at the specified `INDEX`.
+* `PREFIX_LETTER` specifies type of detail to be updated, e.g. `n/` for name.
+* `UPDATED_DETAILS` are the new details of type corresponding to `PREFIX_LETTER` that the Transaction is to be updated with.
+****
+
+Examples:
+
+*   `update i/1 | n/Curry Laksa`
+*   `update i/1 | c/11`
+
+Expected output: +
+** Assume 2nd command occurs after 1st*
```

Comment:
There is an additional `*` in front. Is it intentional or accidental?

Reply:
Intentional to emphasize a point i.e. ***Take note of this**, but will remove to avoid misinterpretation when reading adoc syntax.

Reply:
You can consider using admonition from asciidoc. 

E.g. `NOTE: Assume...`

https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/